app/src/components/Grid.svelte

```diff
@@ -1,11 +1,46 @@
 <script>
   export let s = 17.73
+  let pixels = [];
   $: ss = s*5
 
   // reactive console -----------------------------------
   import bindConsole from '../lib/bindConsole'
   
-  window['grid'] = {}
+  window['grid'] = {
+    // maddie was heeerreee
+    // todo : migrate to a canvas api to make it faster (this is fine for now)
```

Comment:
Remove comment about Canvas API. It's not a todo! Your DOM implementation is perfect for this game, as hackability and ease of understanding is far more important than performance. With the DOM implementation, players can inspect element and get more familiar with HTML at the same time ✨